src/features/collection/components/cards/ExperienceModalCard.tsx

```diff
@@ -0,0 +1,211 @@
+'use client';
+
+import Button from '@/components/Button/Button';
+import Tag from '@/components/Tag/Tag';
+import { Flex, Modal, ModalBody, ModalContent, ModalHeader, ModalOverlay } from '@chakra-ui/react';
+import Image from 'next/image';
+import { Capacity } from '../../types';
+import cardImage from '../../../../../public/images/card1.png';
+import { useState } from 'react';
+import TextAreaField from '@/components/Input/TextAreaField/TextAreaField';
+import MotionBox from './MotionBox';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  // Todo: Data
+  period: string;
+  title: string;
+  summaries: string[];
+  capabilities: Omit<Capacity, 'count'>[];
+};
+
+const ExperienceModalCard = (props: Props) => {
+  const { isOpen, onClose, period, title, summaries, capabilities } = props;
+
+  const [isBack, setIsBack] = useState(false);
+
+  const userCapabilitis = capabilities.filter(({ isAi }) => !isAi);
+  const aiCapabilitis = capabilities.filter(({ isAi }) => isAi);
+
+  const aiRecommend =
+    '디자이너로서 개발팀과 각각의 전문성을 최대한 활용하여 높은 퀄리티의 앱을 만들어내기 위해 커뮤니케이션 능력을 뽐내셨군요! 빠른 기간안에 앱 서비스를 런칭해야하는 상황에서 디자인 시스템 제작, 런칭일 정해서 린하게 개발하는 방법을 제의한 것은 프로젝트 관리 능력의 일환이었습니다.';
+
+  const aiRecommendQuestions = [
+    {
+      id: 10,
+      question:
+        'Q. 본인이 팀 프로젝트에서 어려운 의견 충돌 상황을 어떻게 관리했고, 해결책을 도출하는 과정을 설명해주세요.',
+    },
+    {
+      id: 11,
+      question:
+        'Q. 본인이 팀 프로젝트에서 어려운 의견 충돌 상황을 어떻게 관리했고, 해결책을 도출하는 과정을 설명해주세요.',
+    },
+    {
+      id: 12,
+      question: 'Q. 본인이 고객과의 원활한 커뮤니케이션을 통해 성과를 이뤄낸 경험에 대해 알려주세요.',
+    },
+  ];
+
+  const handleFlipClick = () => {
+    setIsBack((isBack) => !isBack);
+  };
+
+  const cardFlippingVariants = {
+    flipInitial: {
+      rotateY: 0,
+      transition: { duration: 0.5 },
+    },
+    flipEnd: {
+      rotateY: 180,
+      transition: { duration: 0.5 },
+    },
+  };
+
+  return (
+    <Modal size="experienceModalCard" isOpen={isOpen} onClose={onClose} isCentered>
+      <ModalOverlay />
+      <ModalContent bg={'none'} shadow={'none'}>
+        <Flex p={0} className="[perspective:1000px]" flexDir={'column'} justify={'flex-start'} w={1149} h={936}>
+          <MotionBox
+            h={'full'}
+            variants={cardFlippingVariants}
+            animate={!isBack ? 'flipInitial' : 'flipEnd'}
+            bg={'white'}
+            rounded={32}>
+            <ModalHeader
+              className={`w-full ${!isBack ? 'bg-gray-50' : ''} rounded-t-[32px] ${
+                isBack && '[transform:rotateY(180deg)]'
+              }`}
+              p={[50, 42, 40, 30]}
+              m={0}>
+              <div className="flex flex-row justify-between mb-[12px]">
+                <span>{period}</span>
+                <Button variant="gray900" size="M" onClick={handleFlipClick}>
+                  {!isBack ? '카드 뒷면보기' : '카드 앞면보기'}
```

Comment:
```suggestion
                  {isBack ? '카드 앞면보기' : '카드 뒷면보기'}
```
개인적으로 안티패턴이라고 생각합니다~